tests/containers/integration-test-local

```diff
@@ -14,6 +14,7 @@ RUN dnf install -y --repo bluechi-rpms \
         bluechi-ctl \
         bluechi-ctl-debuginfo \
         bluechi-selinux \
+        bluechi-coverage \
```

Comment:
Similar to above, I'd pass `WITH_COVERAGE` value as an argument to container build similar as in https://github.com/containers/bluechi/pull/488/files#diff-fe5746460187f3d335a236fa749d7516f1e342a307150d3e7c3f4aec7183203cR3 and  https://github.com/containers/bluechi/pull/488/files#diff-66c84c667aa7801b2592d803c12fd54c2131c7a17dc74518221f9be618f8a571R20
Then we can install `bluechi-coverage` package only when needed